Source/Core/CircleGeometry.js

```diff
@@ -175,5 +175,12 @@ define([
         });
     };
 
+    /**
+     * @private
```

Comment:
Not really. In order to be on terrain a geometry has to implement `createShadowVolume` which is private, so since this is only required for the same situation I made it private as well. In reality a geometry doesn't even need to implement this, it would just use a shadow volume that was huge like before.